pkg/compactor/compactor.go

```diff
@@ -918,6 +918,7 @@ func (c *Compactor) compactUser(ctx context.Context, userID string) error {
 		fetcher,
 		deduplicateBlocksFilter,
 		ignoreDeletionMarkFilter,
+		c.compactorCfg.CompactionInterval,
```

Comment:
They introduced a timeout for syncing metadatas. I just put compaction interval for now